cmds/live.c

```diff
@@ -220,6 +221,11 @@ static int forward_options(struct uftrace_opts *opts)
 	pr_dbg2("connected to agent %d\n", opts->pid);
 
 	status_close = agent_message_send(sfd, UFTRACE_MSG_AGENT_CLOSE, NULL, 0);
+	if (status_close == 0) {
+		status_close = agent_message_read_response(sfd, &ack);
```

Comment:
`s/status_close/status/` ?

Reply:
I'm using `status` to check the agent response to option forwarding (not implemented, I added a placeholder comment). Using it again to check is the connection closes properly would overwrite it. Forwarding option could fail, then the connection is closed properly, and vice-versa. In either case, I consider that a client error.

Reply:
Sounds reasonable.